src/EnergyPlus/ThermalComfort.cc

```diff
@@ -678,9 +681,9 @@ namespace ThermalComfort {
 			RadTemp = CalcRadTemp( PeopleNum );
 			// Use mean air temp for calculating RH when thermal comfort control is used
 			if ( present( PNum ) ) {
-				RelHum = PsyRhFnTdbWPb( MAT( ZoneNum ), ZoneAirHumRat( ZoneNum ), OutBaroPress );
+				RelHum = PsyRhFnTdbWPb( MAT( ZoneNum ), ZoneAirHumRatAvgComf( ZoneNum ), OutBaroPress );
```

Comment:
This was not an issue uncovered in a defect file but from reviewing code and noticing inconsistencies.